src/api/middleware/errorMiddleware.ts

```diff
@@ -12,7 +14,14 @@ class ResponseError extends Error {
   url: string
 
   constructor(clientName: string, response: ConfluenceResponse) {
```

Comment:
I just noticed that there's a typo in a type here. `ConfluenceResponse` should be `ConfluentResponse`. Could you fix that while you're anyway touching this file?